package.json

```diff
@@ -105,6 +105,7 @@
         "dotenv": "^8.2.0",
         "react": "^17.0.2",
         "react-dom": "^17.0.2",
+        "react-swipeable": "^6.2.1",
```

Comment:
why is this library added to package.json in root? The library already exists in traders package.json where you are implementing the wheelpicker component

Reply:
<img width="596" alt="Screenshot 2024-07-22 at 10 39 19 AM" src="https://github.com/user-attachments/assets/d8c70cb3-27d0-49a9-8ec8-dc004a818531">